catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -247,7 +247,8 @@
     /**
      * Sets {@link Attribute} with new attribute.
      *
-     * @param attribute new {@link Attribute} to set
+     * @param attribute new {@link Attribute} to set. If <code>null</code> is passed as the Attribute
```

Comment:
Should this read something more like `If the getValue() call on the passed in attribute returns <code>null</code>, the attribute will be removed from this {@link Metacard}.`